Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG APP_INSIGHTS_AGENT_VERSION=2.5.1-BETA
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
+FROM hmctspublic.azurecr.io/base/java:17-distroless
```

Comment:
Neha - can you change this to hmctspublic.azurecr.io/base/java:11-distroless. The upgrade to use Java 17 will come under a separate PET ticket.

Reply:
Oh! Will do that!